examples/program_analyzer/client/BalProgramAnalyzer_client.bal

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/lang.value;
+
+public function main() returns error? {
+
+    // Change the maximum inbound message size accordingly
+    BalProgramAnalyzerClient ep = check new ("http://localhost:8981", maxInboundMessageSize = 424193);
+    string s = check ep->syntaxTree();
```

Comment:
Maybe we can improve the variable names
```suggestion
    string syntaxTreeString = check ep->syntaxTree();
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/636/commits/b3f4e50a25ee0e28a1ac8d0eaa008b2313e18e2c